src/lib/layers/editable-geojson-layer.js

```diff
@@ -11,9 +11,9 @@ import destination from '@turf/destination';
 import bearing from '@turf/bearing';
 import pointToLineDistance from '@turf/point-to-line-distance';
 import { point, featureCollection as fc } from '@turf/helpers';
-import type { GeoJsonFeature } from '../../types';
-import { EditableFeatureCollection } from '../editable-feature-collection';
-import EditableLayer from './editable-layer';
+import type { Feature } from '../../geojson-types.js';
```

Comment:
Remove .js ! (Linter did not complain?)

Reply:
In my opinion (and several others), having `.js` is more standards-compliant.

Examples:

* Chrome requires the `.js` extension when using `<script type="module"></script>` [reference](https://stackoverflow.com/a/44481876/444917)
* Ryan Dahl regretted allowing omitting the extension for node.js requires [video](https://youtu.be/M3BM9TB-8yA?t=13m57s)

So, unless there's a compelling reason to leave them off, I'd vote to keep extensions.

Reply:
OK.